spec/requests/lobbies_spec.rb

```diff
@@ -327,4 +333,101 @@
       end
     end
   end
+
+  describe 'players_done' do
+    let(:quiz) { FactoryBot.create(:quiz) }
+    let(:lobby) { FactoryBot.create(:lobby, quiz: quiz) }
+    let(:question) { FactoryBot.create(:question, quiz: quiz) }
+
+    let(:player1) { FactoryBot.create(:player, lobby: lobby) }
+    let(:player2) { FactoryBot.create(:player, lobby: lobby) }
+    let(:player3) { FactoryBot.create(:player, lobby: lobby) }
+
+    let(:answer1) { FactoryBot.create(:answer, question: question) }
```

Comment:
Same suggestion here - no need to use `let`s for things that you won't be referencing in the future. It's a bit awkward to have these `let`s here, and then just "calling" them in `before` to make them execute. (a solution would be to use `let!` which executes its block instantly, but I find that to be a sign of code that can be refactored)